javatests/google/registry/flows/EppFlowInjectionModule.java

```diff
@@ -0,0 +1,46 @@
+package google.registry.flows;
+
+import org.joda.time.DateTime;
+
+import dagger.Module;
+import dagger.Provides;
+import google.registry.flows.EppException.UnimplementedOptionException;
+import google.registry.flows.domain.ExtraDomainValidation;
+import google.registry.model.domain.launch.LaunchCreateExtension;
+import google.registry.model.registry.label.ReservationType;
+
+@Module
+public class EppFlowInjectionModule {
+
+  private final String labelStr = "extra";
```

Comment:
Make these two static as well as final, and use UPPER_CAMEL.  Also use a better name that describes what they do.  Specifically what they are doing is that they cause an extra domain validation error if the incoming domain is equal to this, so how about calling them INVALID_LABEL_EXTRA_VALIDATION and INVALID_TLD_EXTRA_VALIDATION?  Or you could just inline them at all call sites; they're only used twice each and it's just a test.

Also, the value itself could be something clearer.  It's not immediately obvious why "extra.tld" would be a disallowed domain name in the extra logic.  If it were something like "disallowed.tld" that'd make the test more self-documenting.  (You'd also have to change the EPP XML files).